randovania/games/dread/gui/preset_settings/__init__.py

```diff
@@ -20,6 +23,9 @@ def dread_preset_tabs(editor: PresetEditor, window_manager: WindowManager):
 
     return [
         PresetTrickLevel,
+        *([
+            PresetElevatorsDread,
+        ] if randovania.is_dev_version() else []),
```

Comment:
don't even bother tbh. we can add this check if it turns out it's not stable enough in time for 6.2

Reply:
We have the choice of missing it when we want to make it stable or missing to deactivate it, if it shouldn't go to stable :D